### PR TITLE
fix: add dynamic TradingView volume visibility

### PR DESCRIPTION
**Pull request by limichange**
## Summary
- Add app-side Volume visibility events for TradingView K-line history and realtime updates.
- Stop statically disabling Volume in the Swap K-line desktop chart so the chart can decide dynamically.
- Add focused tests for all-zero volume hiding and realtime restore behavior.

## Intent & Context
Swap K-line charts can receive data where every volume value is zero. The requested behavior is to notify the TradingView chart so it hides the Volume study only for those all-zero datasets, while restoring Volume when later data contains non-zero volume.

## Root Cause
Volume was previously disabled statically for the Swap K-line desktop chart. That prevented the chart from distinguishing between tokens that genuinely have volume data and tokens whose volume data is all zero. App-side K-line responses also did not emit a dedicated visibility signal for the chart to react to.

## Design Decisions
- History responses may hide Volume only for firstDataRequest, because the initial snapshot is representative enough to decide whether the chart should start without Volume.
- Realtime single-candle zero-volume updates never hide Volume, because a single zero candle should not remove an existing valid Volume pane.
- Realtime non-zero updates still send hidden:false so the chart can restore Volume after data becomes available.
- Hyperliquid/perps data is handled in the chart repository because that datafeed runs inside the TradingView WebView context, not in the app main runtime.

## Changes Detail
- Added volumeVisibilityHandler for app-to-chart Volume visibility events.
- Wired K-line history, polling realtime, and websocket realtime paths to emit visibility updates.
- Removed the static Volume disabled feature from Swap K-line desktop config.
- Added unit tests for hidden state detection, bridge payloads, and realtime zero-volume suppression.

## Risk Assessment
- **Risk Level**: Medium
- **Affected Platforms**: Mobile / Desktop / Web
- **Risk Areas**: TradingView WebView bridge timing and K-line data source differences between app-served market data and chart-served Hyperliquid data.

## Test plan
- [x] node_modules/.bin/jest packages/kit/src/components/TradingView/TradingViewV2/messageHandlers/volumeVisibilityHandler.test.ts --runInBand
- [x] yarn tsc:staged
- [x] npx oxlint --tsconfig ./tsconfig.json --type-aware touched app files --deny-warnings
- [x] git diff --check

Related chart repo PR: https://github.com/OneKeyHQ/tradingview-charting-library/pull/176

### CODE REVIEW

**Comment by revan-zhang** [comment]
### :white_check_mark: **Snyk checks have passed. No issues have been found so far.**

| Status | Scan Engine |  Critical |  High |  Medium |  Low | Total (0) |
|---|---|---|---|---|---|---|
| :white_check_mark: | Open Source Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/rayston92/pr-checks/f78c49ea-34be-46e9-a79d-9933c6c5bbc4?refs=971012c53b4bf814120c0f1c5adeea7f8c63b131&source=prChecksComment) |
| :white_check_mark: | Licenses| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/rayston92/pr-checks/f78c49ea-34be-46e9-a79d-9933c6c5bbc4/license?refs=971012c53b4bf814120c0f1c5adeea7f8c63b131&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

File: packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx [hunk 1]
```diff
@@ -81,7 +81,6 @@ const SWAP_KLINE_DESKTOP_DISABLED_TRADING_VIEW_FEATURES = [
   TRADING_VIEW_DISABLED_FEATURES.FULLSCREEN,
   TRADING_VIEW_DISABLED_FEATURES.LAYOUT_TOGGLE,
   TRADING_VIEW_DISABLED_FEATURES.DRAWING_TOOLBAR,
-  TRADING_VIEW_DISABLED_FEATURES.VOLUME,
 ] as const satisfies readonly ITradingViewDisabledFeature[];
```
Comment (sidmorizon) [inline packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx:84]: **Volume visibility relies on WebView-side handler that isn't in this diff**

Type: `Flag`

The removal of `TRADING_VIEW_DISABLED_FEATURES.VOLUME` from `SWAP_KLINE_DESKTOP_DISABLED_TRADING_VIEW_FEATURES` at `packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx:83` means volume is no longer statically hidden in swap KLine views. Instead, the new `VOLUME_VISIBILITY_UPDATE` message (`packages/kit/src/components/TradingView/TradingViewV2/messageHandlers/volumeVisibilityHandler.ts:48`) is sent to the WebView to dynamically control visibility. However, the WebView-side JavaScript handler that receives and acts on `VOLUME_VISIBILITY_UPDATE` is not included in this diff. If the WebView handler is missing or not yet deployed, volume would always be visible in swap KLine — a regression from the previous always-hidden behavior. This should be confirmed externally.

Comment (limichange) [reply packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx:84]: Confirmed. The WebView-side handler is implemented in the paired chart repo PR:
https://github.com/OneKeyHQ/tradingview-charting-library/pull/176

This app PR only sends the `VOLUME_VISIBILITY_UPDATE` event and removes the static Volume disable flag; the chart PR consumes the event and applies dynamic Volume hide/restore.

**Review by zhaono1 — APPROVED** [review]

**Review by originalix — APPROVED** [review]
